src/Learners/ClassifierModels/Properties/launchSettings.json

```diff
@@ -2,7 +2,7 @@
   "profiles": {
     "ClassifierModels": {
       "commandName": "Project",
-      "commandLineArgs": "."
+      "commandLineArgs": "\"C:\\Users\\jonat\\source\\repos\\TrueSkill2\\Infer.NET\\src\\Learners\\Classifier\\BayesPointMachineClassifierInternal\\GeneratedAlgorithms\""
```

Comment:
This is odd.

Reply:
Sorry, this was for debugging. Will remove.